### DISCUSSION TITLE
Update KitQL docs

### DISCUSSION THREAD

**Discussion by Parables** [jycouet/kitql#214] [Docs Discussion] [answered]
SvelteKit is changing and hence, the docs needs to be updated and versioned appropriately

**Comment by jycouet** [comment, marked as answer]
I will soon 👍

**Comment by Parables** [comment]
Sincerely grateful 🙏

**Reply by Parables** [reply]
Any updates on this?

**Reply by jycouet** [reply]
Let me know what it not up-to-date. 
Everything should be in sync and working now

**Reply by Parables** [reply]
That's great... I'm taking inspiration from KitQL to provide a similar DevX with Flutter.

Thank you so much for KitQL
